public_suffix_list.dat

```diff
@@ -11209,6 +11209,12 @@ zuerich
 // ===BEGIN PRIVATE DOMAINS===
 // (Note: these are in alphabetical order by company name)
 
+// 12CHARS: https://12chars.com
+// Submitted by Kenny Niehage <kenny@12chars.com>
```

Comment:
If possible, replace this with a role based address so it stays reachable when you leave the company.

Reply:
We have implemented a PSL-specific e-mail address to implement this suggestion.